tests/playwright-test/loader.spec.ts

```diff
@@ -104,3 +104,35 @@ test('should validate configuration object', async ({ runInlineTest }) => {
   expect(result.failed).toBe(0);
   expect(result.output).toContain('playwright.config.ts: config.timeout must be a non-negative number');
 });
+
+test('should load an mjs file', async ({ runInlineTest }) => {
+  // modules don't work with node < 12
+  test.skip(parseInt(process.versions.node, 10) < 12);
+  const { exitCode, passed } = await runInlineTest({
+    'a.spec.mjs': `
+        const { test } = pwt;
+        test('succeeds', () => {
+          expect(1 + 1).toBe(2);
+        });
+      `
+  });
+  expect(passed).toBe(1);
+  expect(exitCode).toBe(0);
+});
+
+test('should throw a nice error if a js file uses import', async ({ runInlineTest }) => {
+  // modules don't work with node < 12
+  test.skip(parseInt(process.versions.node, 10) < 12);
```

Comment:
We never run tests with Node.js < 12 and we only support 12+. So we can remove that.